test/topology/test_topology.py

```diff
@@ -61,3 +66,69 @@ async def test_decommission_node_add_column(manager, random_tables):
     await manager.decommission_node(servers[1])             # Decommission [1]
     await table.add_column()
     await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+async def test_remove_node_with_concurrent_ddl(manager, random_tables):
+    stopped = False
+    ddl_failed = False
+
+    async def do_ddl():
+        nonlocal ddl_failed
+        iteration = 0
+        while not stopped:
+            logger.debug(f'ddl, iteration {iteration} started')
+            try:
+                # If the node was removed, the driver may retry "create table" on another node,
+                # but the request might have already been completed.
+                # The same applies to drop_table.
+
+                await random_tables.add_tables(5, 5, if_not_exists=True)
+                await random_tables.verify_schema()
+                while len(random_tables.tables) > 0:
+                    await random_tables.drop_table(random_tables.tables[-1], if_exists=True)
+                logger.debug(f'ddl, iteration {iteration} finished')
+            except:
+                logger.exception(f'ddl, iteration {iteration} failed')
+                ddl_failed = True
+                raise
```

Comment:
Maybe catch the exception and communicate the test failed with something better than a stack trace?

Reply:
Why hide the stack trace? It usually indicates the cause of the problem.

Reply:
you can catch known exceptions and `raise ... from exc`

Reply:
Also, since we wait for `do_remove_node()` first, what will happen both `do_ddl` `do_remove_node` raise? We won't await on `ddl_task`. Will we see the exception from `do_ddl`?

While there... perhaps we should wait on `ddl_task` in a `finally`, so we don't leave background operations running when exiting the test?

Reply:
good point, will move `await ddl_task` to finally